superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -606,7 +692,12 @@ export const FiltersConfigForm: React.FC<FiltersConfigFormProps> = ({
                           });
                           forceUpdate();
                         }}
-                        label={<StyledLabel>{t('Adhoc filters')}</StyledLabel>}
+                        label={
+                          <span>
+                            <StyledAsterisk>*</StyledAsterisk>
```

Comment:
Nit: it would feel more intuitive if this were just `<StyledAsterisk />` without having to explicitly put in the asterisk as a child (I assume we don't expect to put in other characters in it).